Apps/AdminWebClient/src/Server/Delegates/RestImmunizationAdminDelegate.cs

```diff
@@ -178,6 +186,24 @@ public async Task<RequestResult<PHSAResult<IList<ImmunizationViewResponse>>>> Ge
             return retVal;
         }
 
+        /// <summary>
+        /// Determines if the Immunization records are being refreshed and waits if true.
+        /// </summary>
+        /// <param name="result">The response from the Immunization delegate.</param>
+        /// <returns>true if a retry should be attempted by the delegate.</returns>
+        private bool Retry(RequestResult<PHSAResult<IList<ImmunizationViewResponse>>> result)
```

Comment:
nit Retry sounds like this method does the action. A name like ShouldRetry sounds better